cmd/docs.go

```diff
@@ -57,11 +58,14 @@ func docsMain(cmd *cobra.Command, args []string) exitCode {
 	}
 
 	// Print the supported attributes. This part is printed as
-	//   SUPPORTED_ATTRIBUTES
+	//   SUPPORTED ATTRIBUTES
 	//     * <attribute> (<full_name_of_attribute>)
 	//
 	//   <description that talks about attributes/metadata and shows off 'meta'>
-	if len(entry.Attributes.ToMap()) > 0 {
+	//
+	// if the entry has any supported attributes. Otherwise, it prints an appropriate
+	// note then prints the attributes/metadata description.
+	if len(entry.Metadata) > 0 {
```

Comment:
Could this go wrong if the entry has some attributes but no metadata?

Reply:
It looks like if that's true, we'll return attributes as metadata. And if there is metadata, you specifically get attributes. So seems like this will work but is slightly confusing.

Reply:
Ah yeah, I'll add a clarifying comment. Thanks for bringing it up,

Reply:
Updated.